app/javascript/mastodon/hooks/useLinks.ts

```diff
@@ -12,7 +12,7 @@ const isMentionClick = (element: HTMLAnchorElement) =>
   !element.classList.contains('hashtag');
 
 const isHashtagClick = (element: HTMLAnchorElement) =>
-  element.textContent?.[0] === '#' ||
+  element.textContent.startsWith('#') ||
```

Comment:
Can we have an issue if `textContent` is `undefined`? We had `?.` before.

Same in a few other places